src/dbnode/server/server.go

```diff
@@ -666,6 +666,20 @@ func Run(runOpts RunOptions) {
 
 	opts = opts.SetBootstrapProcessProvider(bs)
 	timeout := bootstrapConfigInitTimeout
+
+	bsGauge := instrument.NewStringListEmitter(scope, "bootstrapper_bootstrappers", "bootstrapper")
```

Comment:
Maybe just call it "bootstrappers" instead of "bootstrapper_bootstrappers" to make it a little more simple?

Reply:
Yeah I agree. The only reason why I had it `bootstrapper_bootstrappers` in the first place was because there is another metric which is starting with `bootstrapper_` (forgot which) so I thought I could piggy-back on that namespace. I changed it to just `bootstrappers` now.